src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchXpathFilterTest.java

```diff
@@ -42,9 +42,6 @@ public void testIllegalToken() throws Exception {
     }
 
     @Test
-    @Ignore("MethodLength should have a violation when method length"
-            + " is more than threshold, but not. It should be solved by"
-            + "context strategy.")
     public void testMethodLength() throws Exception {
```

Comment:
Why violation is expected in newline and patchedline strategies - `Test.java:4:5: Method length is 5 lines (max allowed is 3).` ? This should be expected only on context strategy.
Also, you should to add `MethodLength/context/***` files and add another `testByConfig()`

Reply:
done